internal/dataplane/kong_client.go

```diff
@@ -484,6 +485,7 @@ func (c *KongClient) sendToClient(
 	)
 
 	c.recordResourceFailureEvents(entityErrors, KongConfigurationApplyFailedEventReason)
+	c.recordApplyConfigurationEvents(err)
```

Comment:
Can we include the client address in the event as well? The information is readily available in scope of this function.

Reply:
done.